sqlserver/check.py

```diff
@@ -324,10 +382,12 @@ def open_db_connections(self, instance):
         ]
 
         try:
-            rawconn = adodbapi.connect(self._conn_string(instance=instance),
-                                       timeout=timeout)
-            self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK,
-                               tags=service_check_tags)
+            if self._get_connector(instance) == 'adodbapi':
```

Comment:
Yep, it's really not important, I just noticed that sometimes you do a `.lower` on the connector and not there, but people using the connector field will probably want the odbc one anyway.